<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoGenerator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="B1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="page_num" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="B2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="page_num" stdset="0">
     <number>2</number>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>20</width>
      <height>351</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>541</width>
      <height>341</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>欢迎！
当前目标：雅虎（日本）新闻，请选择新闻类别</string>
      </property>
     </widget>
     <widget class="QComboBox" name="P1_CB_CAT">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>251</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>请选择新闻：</string>
      </property>
     </widget>
     <widget class="QListWidget" name="P1_LW_NEWS">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>531</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="P1_B_SELECT">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>90</y>
        <width>80</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>修改新闻翻译页</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3"/>
   </widget>
   <widget class="QPushButton" name="B3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="page_num" stdset="0">
     <number>3</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVideo_Generator">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <addaction name="menuVideo_Generator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
